src/main/java/uk/gov/hmcts/ccd/data/caseaccess/SwitchableCaseUserRepository.java

```diff
@@ -27,7 +27,7 @@
 
     public SwitchableCaseUserRepository(@Qualifier(CachedCaseDetailsRepository.QUALIFIER) final CaseDetailsRepository caseDetailsRepository,
                                         @Qualifier(CCDCaseUserRepository.QUALIFIER) final CaseUserRepository ccdCaseUserRepository,
-                                        @Qualifier(AMCaseUserRepository.QUALIFIER) final CaseUserRepository amCaseUserRepository,
+                                        @Qualifier(AMCaseUserRepository.ACCESS_MANAGEMENT_QUALIFIER) final CaseUserRepository amCaseUserRepository,
```

Comment:
Again why away from convention?

Reply:
Changed to AMCaseUserRepository.QUALIFIER.